File: cs50x/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Change owed: ");
    }
    while (n < 0);

    int coins=0;
    coins+= n/25;
    n=n % 25;
    coins+= n/10;
    n=n % 10;
    coins+= n/5;
    n=n % 5;
    coins+=n;

    printf("%d\n",coins);
}

File: cs50x/sentimental-cash/cash.py

import cs50

change = cs50.get_float("Change: ")
while(change < 0):
    change = cs50.get_float("Change: ")
coins = 0
change = int (100 * change)

coins += change // 25
change %= 25

coins += change // 10
change %= 10

coins += change // 5
change %= 5

coins += change

print(coins)
